js/charts/infographics.js

```diff
@@ -0,0 +1,601 @@
+     */
+    var _x = d3.scalePoint(),
+        _y = d3.scaleLinear(),
+        _line = d3.line(),
+        _area = d3.area();
+
+    /* -------------------------------------------------------------------------------- */
+    var _setConfigParams = function(config) {
+        this.dimension(config.dimension);
+        this.measure(config.measure);
+        this.chartType(config.chartType);
+        this.chartDisplayColor(config.chartDisplayColor);
+        this.chartBorderColor(config.chartBorderColor);
+        this.kpiDisplayName(config.kpiDisplayName);
+        this.kpiAlignment(config.kpiAlignment);
+        this.kpiBackgroundColor(config.kpiBackgroundColor);
+        this.kpiNumberFormat(config.kpiNumberFormat);
+        this.kpiFontStyle(config.kpiFontStyle);
+        this.kpiFontWeight(config.kpiFontWeight);
+        this.kpiFontSize(config.kpiFontSize);
+        this.kpiColor(config.kpiColor);
+        this.kpiColorExpression(config.kpiColorExpression);
+        this.kpiIcon(config.kpiIcon);
+        this.kpiIconFontWeight(config.kpiIconFontWeight);
+        this.kpiIconColor(config.kpiIconColor);
+        this.kpiIconExpression(config.kpiIconExpression);
+        this.tooltip(config.tooltip);
+    }
+
+    var _getKpiDisplayName = function() {
+        if(_kpiDisplayName.trim() == '') {
+            return _measure;
+        }
+
+        return _kpiDisplayName;
+    }
+
+    var _getKpi = function(value, endValue) {
+        var numberOutput = "",
+            iconOutput = "";
+
+        var style = {
+            'font-style': _kpiFontStyle || COMMON.DEFAULT_FONTSTYLE,
+            'font-weight': _kpiFontWeight || COMMON.DEFAULT_FONTWEIGHT,
+            'font-size': _kpiFontSize || COMMON.DEFAULT_FONTSIZE,
+            'color': _kpiColor || COMMON.DEFAULT_COLOR
+        };
+
+        if(_kpiColorExpression) {
+            style['color'] = UTIL.expressionEvaluator(_kpiColorExpression, endValue, 'color');
+        }
+
+        style = JSON.stringify(style);
+        style = style.replace(/["{}]/g, '').replace(/,/g, ';');
+
+        numberOutput += "<span style='" + style + "'>"
+            + UTIL.getNumberFormatterFn(_kpiNumberFormat)(UTIL.roundNumber(value, 0)).toUpperCase()
+            + "</span>";
+
+        var iconStyle = {
+            'font-weight': _kpiIconFontWeight || COMMON.DEFAULT_FONTWEIGHT,
+            'color': _kpiIconColor || COMMON.DEFAULT_COLOR,
+            'font-size': _kpiFontSize || COMMON.DEFAULT_FONTSIZE
+        };
+
+        if(_kpiIconExpression) {
+            _kpiIcon = UTIL.expressionEvaluator(_kpiIconExpression, endValue, 'icon');
+            iconStyle['color'] = UTIL.expressionEvaluator(_kpiIconExpression, endValue, 'color');
+        }
+
+        iconStyle = JSON.stringify(iconStyle);
+        iconStyle = iconStyle.replace(/["{}]/g, '').replace(/,/g, ';');
+
+        iconOutput += "<i class=\"" + _kpiIcon + "\" style=\"" + iconStyle + "\" aria-hidden=\"true\"></i>";
```

Comment:
Same here!

Reply:
Same reason as above, to make the HTML human readable and due to the presence of multiple variables.